app/components/gcse_qualification_cards_component.rb

```diff
@@ -47,14 +53,19 @@ def naric_statement(qualification)
   end
 
   def grade_details(qualification)
-    if qualification.subject == ApplicationQualification::SCIENCE_TRIPLE_AWARD
+    case qualification.subject
+    when ApplicationQualification::SCIENCE_TRIPLE_AWARD
       grades = qualification.structured_grades
       [
         "#{grades['biology']} (Biology)",
         "#{grades['chemistry']} (Chemistry)",
         "#{grades['physics']} (Physics)",
       ]
-    elsif qualification.structured_grades
+    when ApplicationQualification::SCIENCE_DOUBLE_AWARD
+      ["#{qualification.grade} (Double award)"]
+    when ApplicationQualification::SCIENCE_SINGLE_AWARD
+      ["#{qualification.grade} (Single award)"]
+    when ->(_n) { qualification.structured_grades }
       grades = JSON.parse(qualification.structured_grades)
       grades.map { |k, v,| "#{v} (#{k.humanize.titleize})" }
```

Comment:
This causes double award to upcase. We should probably have a bespoke bit of logic for when grade.key == english_double_award 

![image](https://user-images.githubusercontent.com/42515961/100874332-51fcda00-349c-11eb-9b45-3b279343ad7f.png)